rest/admin_api.go

```diff
@@ -840,6 +841,13 @@ func (h *handler) handlePutDbAuditConfig() error {
 		}
 		return nil
 	})
+	if err != nil {
+		base.Audit(h.ctx(), base.AuditIDAuditConfigChanged, base.AuditFields{
+			"audit_scope":          "db",
+			base.AuditFieldPayload: body,
+		})
+	}
+	return err
```

Comment:
This construction is backward:

```suggestion
	if err != nil {
		return err
	}
	base.Audit(h.ctx(), base.AuditIDAuditConfigChanged, base.AuditFields{
		"audit_scope":          "db",
		base.AuditFieldPayload: body,
	})
	return nil
```